msteams-platform/concepts/authentication/authentication.md

```diff
@@ -11,12 +11,17 @@ Authentication is all about validating app users, and securing the app and app u
 
 Choose to add authentication for your app in one of the two ways:
 
-- **Enable single sign-on (SSO) in a Teams app**:
+* **Enable single sign-on (SSO) in a Teams app**:
   SSO within Teams is an authentication method that uses an app user's Teams identity to provide them access to your app. A user who has logged into Teams doesn't need to log in again to your app within the Teams environment. With only a consent required from the app user, the Teams app retrieves access details for them from Microsoft Entra ID. After the app user has given consent, they can access the app even from other devices without having to be validated again.
 
-- **Enable authentication using third-party OAuth provider**:
+* **Enable authentication using third-party OAuth provider**:
   You can use a third-party OAuth Identity Provider (IdP) to authenticate your app users. The app user is registered with the identity provider, which has a trust relationship with your app. When the user attempts to log in, the identity provider validates the app user and provides them access to your app. Microsoft Entra ID is one such third party OAuth provider. You can use other providers, such as Google, Facebook, GitHub, or any other provider.
 
+* **Nested Authentication**: Nested App Authentication (NAA) enables SSO for applications nested (embedded) inside of supported first-party applications. Compared with existing full-trust authentication models, and the on-behalf-of flow, NAA provides better security and greater flexibility in app architecture, enabling the creation of rich, client-driven applications. NAA enables developers to use MSAL.js to acquire tokens for their Teams Tab or Office add-in application, using capabilities of the first-party host to enable Conditional Access and Intune policies (For example, registered device policy) which couldn't otherwise be supported when hosted inside of native WebViews. NAA is supported in MSAL.js v3.1 and higher.
+
+> [!NOTE]
+> Nested app authentication is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
```

Comment:
I don't know the best way to message this, it might as you already have it, but this is Teams R3.6, which I am told can be accessed by any of these options:

Targeted Release - https://learn.microsoft.com/en-us/microsoft-365/admin/manage/release-options-in-office-365?view=o365-worldwide
Public Preview - https://learn.microsoft.com/en-us/microsoftteams/public-preview-doc-updates?tabs=new-teams-client
Developer Preview - https://learn.microsoft.com/en-us/microsoftteams/platform/resources/dev-preview/developer-preview-intro?tabs=new-teams-client#desktop-or-web-client

Reply:
For Teams ring 3.6 we usually update it as Developer Preview.

For Public Preview and Targeted Release we usually don't release the docs to live.